content/en/apps/examples/anc.md

```diff
@@ -79,21 +86,96 @@ Both maternal and newborn care workflows are defined to connect form actions and
+        task="A [child referral follow up task](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.gwiatej2wjp) appears 3 days after the assessment, and lasts for 3 days." 
+        resolution="The CHW submits a child referral follow up form confirming that the patient has attended the clinic visit."
+  %}}
+  {{% workflow
+        condition="A child assessment form doesn’t trigger a referral, yet has one of the following condition:<ul>          <li> fever that lasts less than 7 days and RDT that is positive          <li> fever that lasts less than 7 days and with a RDT that is negative or RDT not done          <li> cough less than 14 days, without fast breathing          <li> diarrhea that lasts less than 14 days and with no blood          <li> yellow MUAC</ul>"
+        task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.9sln0fedxe4x) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+        resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+  %}}
+  {{% workflow
+        condition="A child assessment form doesn’t trigger a referral or treatment, yet has a red MUAC."
+        task="A [malnutrition follow up task](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) appears after 3 days. The task persists 3 days after the due date."
+        resolution="The CHW submits a malnutrition follow up form confirming that the patient has attended the clinic visit." 
+  %}} 
+{{% /workflow-table %}} 
+
+## Supplement schedule
+
+{{% schedule %}}
+|| 6m | 12m | 18m | 2y | 2.5y | 3y | 3.5y | 4y | 4.5y | 5y |
+|------------|--|--|--|--|--|--|--|--|--|--|
+| Deworming  | | X  | X | X | X | X | X | X | X | X |
+| Vitamin A  | X | X | X | X | X | X | X | X | X | X |
+{{% /schedule %}}
+
+## Immunization schedule
+
+{{% schedule %}}
+||birth | 6w | 10w | 14w | 6m | 9m | 12m | 18m |         
+|-----------------------------------|----|----|---|----|--|---|--|---|        
+| BCG                               | X  |    |   |    |  |   |  |   |       
+| Oral Polio Vaccine (OPV)          | X  | X  | X | X  |  |   |  |   |       
+| RotaVirus vaccine                 |    | X  | X |    |  |   |  |   |        
+| Pneumococcal vaccine              |    | X  | X | X  |  |   |  |   |       
+| Pentavalent vaccine               |    | X  | X | X  |  |   |  |   |        
+| Inactivated Polio Vaccine (IPV)   |    |    |   | X  |  |   |  |   |        
+| Measles vaccine                   |    |    |   |    |  | X |  | X |        
+| Yellow fever vaccine              |    |    |   |    |  | X |  |   |
+{{% /schedule %}}
+
```

Comment:
@maremegaye  - I see the 9mo column is empty.  Is this by design to denote we're skipping the 9mo interval or should we just remove this to simplify the schedule?

[<img src="https://user-images.githubusercontent.com/8253488/107838377-7a396c80-6d5a-11eb-811c-34c319472390.png" width=40% height=40%>](https://user-images.githubusercontent.com/8253488/107838377-7a396c80-6d5a-11eb-811c-34c319472390.png)

cc @abbyad

Reply:
@mrjones-plip good point! I removed the empty cells (6 and 12m) and pushed the modifications

Reply:
Looks good - thanks!